### ISSUE TITLE
[Feature] Add new Component section named as Social Buttons 

### ISSUE THREAD

**Issue by SaiyamTuteja** [Rakesh9100/Beautiify#990]
### Description

I want to add the new component section named as Social Buttons and also add the button in the social button page and add new buttons in it 

**Assign this issue under Gssoc2024** 

### Screenshots

> 1

![image](https://github.com/Rakesh9100/Beautiify/assets/119167105/bb9c8971-aad2-4b89-a640-c0cf2e290a18)
### 

> 2

![image](https://github.com/Rakesh9100/Beautiify/assets/119167105/e21ccadc-3fcb-4732-ae2a-860c7c4a37c2)
### 

> 3

![image](https://github.com/Rakesh9100/Beautiify/assets/119167105/417069a6-a20c-4c71-bf7f-9118852ee79e)
### 

> 4

![image](https://github.com/Rakesh9100/Beautiify/assets/119167105/754fe046-09cd-416f-920c-7dc55b738225)

### Checklist

- [X] I have checked the existing issues.
- [X] I follow [Contributing Guidelines](https://github.com/Rakesh9100/Beautiify/blob/main/.github/CONTRIBUTING_GUIDELINES.md) of this project.
- [X] I am a GSSOC'24 contributor.
- [ ] I am a IWOC'24 contributor.
- [ ] I am a JWOC'24 contributor.
- [ ] I am a SWOC'24 contributor.

### Code of Conduct

- [X] I agree to follow this project's "[Code of Conduct](https://github.com/Rakesh9100/Beautiify/blob/main/.github/CODE_OF_CONDUCT.md)".

**Comment by Rakesh9100** [comment]
Is there a need of this separate category as these buttons can come under the Buttons Category @SaiyamTuteja 

Provide your opinion on this @saurabhsingh720

**Comment by saurabhsingh720** [comment]
@Rakesh9100 
Yes using separate component it is easy to identify the type of button, wheather the button is social or normal

**Comment by SaiyamTuteja** [comment]
@Rakesh9100 can I proceed on it

**Comment by Rakesh9100** [comment]
Okay, carry on to add a new category `Social-Buttons` and add at least 2 components in that @SaiyamTuteja 

Provide me the proper name and images of the 2 components you will add inside that category!

**Comment by Rakesh9100** [comment]
Provide the names such that the name contains the prefix `Social-Button` @SaiyamTuteja

**Comment by SaiyamTuteja** [comment]
1. Social Button for LinkedIn 
2. Social button for share 
3. Social button for GitHub , Twitter etc.

**Comment by Rakesh9100** [comment]
Sorry, provide me the names with the suffix Social-Button @SaiyamTuteja

**Comment by SaiyamTuteja** [comment]
### like this 

1. LinkedIn Social Button  
2.  Share Social button 
3. GitHub , Twitter etc. Social button 

If not then pls give one example of it or an idea of it so that i can make changes into it.
